addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -174,7 +175,8 @@
 </td></tr>
 <!-- POWERED BY -->
 <tr><td align="center" style="min-width: 590px;">
-        Powered by <a target="_blank" href="https://www.odoo.com?utm_source=db&amp;utm_medium=email" style="color: #875A7B;">Odoo</a>
+        Powered by <a target="_blank" href="https://www.odoo.com?utm_source=db&amp;utm_medium=email"
+                      t-att-style="'color: ' + (company.email_secondary_color or '#875A7B') + ';'">Odoo</a>
```

Comment:
After seeing this line so many times, starting to think it would be nice to have a small method on "company" that returns its value or the fallback with the ";" 😄